diff --git a/includes/commerce_shipping.checkout_pane.inc b/includes/commerce_shipping.checkout_pane.inc
index 872260f..a6badcd 100644
--- a/includes/commerce_shipping.checkout_pane.inc
+++ b/includes/commerce_shipping.checkout_pane.inc
@@ -92,7 +92,7 @@ function commerce_shipping_pane_checkout_form($form, &$form_state, $checkout_pan
       $order_wrapper = entity_metadata_wrapper('commerce_order', $order);
 
       foreach ($order_wrapper->commerce_line_items as $delta => $line_item_wrapper) {
-        if ($line_item_wrapper->type->value() == 'shipping' && !empty($options[$line_item_wrapper->commerce_shipping_service->value()])) {
+        if ($line_item_wrapper->value() && $line_item_wrapper->type->value() == 'shipping' && !empty($options[$line_item_wrapper->commerce_shipping_service->value()])) {
           $default_value = $line_item_wrapper->commerce_shipping_service->value();
 
           // If the service details value is empty, see if there are any values
